test/unit/models/classes/LtiProvider/ConfigurableLtiProviderRepositoryTest.php

```diff
@@ -101,7 +101,7 @@ public function testConstructorWithNullProviderListThrowsException()
         $subject = new ConfigurableLtiProviderRepository([
             ConfigurableLtiProviderRepository::OPTION_LTI_PROVIDER_LIST => null
         ]);
-        $this->setExpectedException(InvalidArgumentException::class);
+        $this->expectException(InvalidArgumentException::class);
```

Comment:
I'm worried that this will be fixed in separate PR related to upgrading code to PHPUnit 7.

Reply:
i think we can drop the PR related to phpunit 7, because we are going to version 8